app/models/ems_event.rb

```diff
@@ -227,6 +227,7 @@ def get_target(target_type)
 
     target_type = "src_vm_or_template"  if target_type == "src_vm"
     target_type = "dest_vm_or_template" if target_type == "dest_vm"
+    target_type = "middleware_server"   if event.event_type == "hawkular_event"
```

Comment:
> Unless the logic have been changed recently the generic hawkular_events were not shown on the timeline, 

No? maybe you are right but I'm not sure why they would not show up.  Do you know?